sim/rogue/backstab.go

```diff
@@ -31,19 +31,19 @@ func (rogue *Rogue) registerBackstabSpell() {
 		BonusCritRating: core.TernaryFloat64(rogue.HasSetBonus(ItemSetVanCleefs, 4), 5*core.CritRatingPerCritChance, 0) +
 			10*core.CritRatingPerCritChance*float64(rogue.Talents.PuncturingWounds),
 		// All of these use "Apply Aura: Modifies Damage/Healing Done", and stack additively (up to 142%).
-		DamageMultiplierAdditive: 1 +
+		DamageMultiplier: 1.5 * (1 +
```

Comment:
Again, why not use the additive field here?

Reply:
Simplicity.